plugins/inputs/ethtool/ethtool.go

```diff
@@ -0,0 +1,61 @@
+package ethtool
+
+import (
+	"sync"
+)
+
+type Command interface {
+	DriverName(intf string) (string, error)
+	InterfaceNames() ([]string, error)
+	Stats(intf string) (map[string]uint64, error)
+}
+
+type Ethtool struct {
+	// This is the list of interface names to include
+	IncludeInterfaces []string `toml:"include_interfaces"`
```

Comment:
Make these proper include/exclude filters:
```go
interface_include = []
interface_exclude = []
```

See [this plugin](https://github.com/influxdata/telegraf/blob/1.11.2/plugins/inputs/kube_inventory/kube_state.go#L88-L97) for reference on making and using the include/exclude filter.

Reply:
Now using the filter as per suggestion